stdlib/3/ast.pyi

```diff
@@ -14,9 +14,21 @@ from typing_extensions import Literal
 from _ast import *  # type: ignore
 
 if sys.version_info >= (3, 8):
-    class Num(Constant): ...
-    class Str(Constant): ...
-    class Bytes(Constant): ...
+    class Num(Constant):
+        value: complex
```

Comment:
Float isn't a subtype of complex, right? In practice it's much more commonly a float or int.

Reply:
I was confused about that too, then I found this on mypy docs: https://mypy.readthedocs.io/en/stable/duck_type_compatibility.html#duck-type-compatibility

it says both `int` and `float` are duck types compatible with `complex`